pil-analyzer/src/pil_analyzer.rs

```diff
@@ -435,16 +435,22 @@ impl PILAnalyzer {
                 vec![]
             }
             PilStatement::Include(_, _) => unreachable!(),
-            _ => {
+            stmt => {
                 let names = statement
                     .symbol_definition_names_and_contained()
                     .map(|(name, sub_name, symbol_category)| {
                         (
                             match sub_name {
-                                None => self.driver().resolve_decl(name),
+                                None => self
+                                    .driver()
+                                    .resolve_decl(name)
+                                    .map_err(|e| stmt.source_reference().with_error(e))
```

Comment:
Is it possible for this to create an error? We are iterating over all defined names in this statement and only call `resolve_decl` to make it absolute - do I understand that correctly? So we can use `unwrap` here.

Reply:
Actually see my new comment on `resolve_decl` - this is not supposed to fail anyway.